package.json

```diff
@@ -14,16 +14,16 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^4.2.4",
-    "@angular/common": "^4.2.4",
-    "@angular/compiler": "^4.2.4",
-    "@angular/core": "^4.2.4",
+    "@angular/animations": "4.3.6",
```

Comment:
Wonder if it is possible to use a variable to keep these in sync like http://ianmcnally.me/blog/2016/2/4/variables-in-npm-scripts

Reply:
This doesn't seem to work, when I do this it thinks the variable name is the version I am trying to install.

Reply:
Rats